docs/index.rst

```diff
@@ -0,0 +1,11 @@
+#############################
+Ephemeris Abstraction Library
```

Comment:
Maybe update this to: `Abstraction Layer for Ephemerides (ALE)` ? (This project was briefly called "EAL" in the past.)

Reply:
Good catch! I just copied up the old index.rst that was in docs/source/ folder (as opposed to just docs/). So I fixed it in there as well!